dl1_data_handler/dl_eventsources.py

```diff
@@ -0,0 +1,302 @@
+        
+        zenith = np.asarray(self.event_M1["MSrcPosCam.fY"].array())
+        
+        pointing_altitude = np.asarray(self.event_M1["MPointingPos.fZd"].array())
+        
+        azimuth = np.asarray(self.event_M1["MSrcPosCam.fX"].array())
+        
+        pointing_azimuth = np.asarray(self.event_M1["MPointingPos.fAz"].array())
+        
+        core_x = np.asarray(self.event_M1["MMcEvt.fCoreX"].array())
+        core_y = np.asarray(self.event_M1["MMcEvt.fCoreY"].array())
+        
+        mc_energy = np.asarray(self.event_M1["MMcEvt.fEnergy"].array())/1000.0
+        h_first_int = np.asarray(self.event_M1["MMcEvt.fZFirstInteraction"].array())
+        
+        mask = r".([A-Z]+)_M\d_za\d+to\d+_\d_\d+_Y_.*"
+        primary_id = re.findall(mask, self.file_list[0])[0]
+        if primary_id == 'GA':
+            shower_primary_id = 1
+            
+        stereo_total = np.max(eventid_M1)
+        event_index = np.zeros(shape = (stereo_total,1))
+        
+        #Reading data from root file for Image table
+        
+        charge_M1 = self.event_M1["MCerPhotEvt.fPixels.fPhot"].array()
+        peak_time_M1 = self.event_M1["MArrivalTime.fData"].array()
+        charge_M1 = np.asarray(charge_M1)
+        peak_time_M1 = np.asarray(peak_time_M1)
+        
+        charge_M2 = self.event_M2["MCerPhotEvt.fPixels.fPhot"].array()
+        peak_time_M2 = self.event_M2["MArrivalTime.fData"].array()
+        charge_M2 = np.asarray(charge_M2)
+        peak_time_M2 = np.asarray(peak_time_M2)
+        
+        total_events = len(self.event_M1["MCerPhotEvt.fPixels.fPhot"].array())
+        #Iterating over all events, and saving only stereo ones
+        tels_in_file = ["m1", "m2"]
+        tels_with_data = {1,2}
+        for i in range(0, total_events):
+            if eventidM1[i] != 0:
+                obs_id = self.run_number
+                event_id = eventid_M1[i]
+                i2 = np.where(eventid_M2==eventid_M1[i])
+                i2 = int(i2[0])
+                data.count = counter
+
+                # Setting up the Data container
+                data.index.obs_id = obs_id
+                data.index.event_id = event_id
+                data.r0.tel.clear()
+                data.r1.tel.clear()
+                data.dl0.tel.clear()
+                
+                # Filling the DL1 container with the event data
+                for tel_i, tel_id in enumerate(tels_in_file):
+                    
+                    #Adding telescope pointing container
+
+                    data.pointing.tel[tel_i+1].azimuth = u.Quantity(np.deg2rad(pointing_azimuth[i]), u.rad)
+                    data.pointing.tel[tel_i+1].altitude = u.Quantity(np.deg2rad(90.0 - pointing_altitude[i]), u.rad)
+                    
+                    
+                    #Adding MC data
+                    data.mc.alt = Angle(np.deg2rad(zenith[i] * 0.00337), u.rad)
+                    data.mc.az = Angle(np.deg2rad(azimuth[i] * 0.00337), u.rad)
```

Comment:
Same here. And we should also rename the column in the table.

Reply:
The column name is defined in writer.py, right? It is same for all telescopes MAGIC, VERITAS, LST, etc.

Reply:
I see, the "EventTableRow" are generically defined in table_definitions.py. I think for the time being, we can leave it like it is. Can you please add a comment here.

Reply:
Sure